pkg/common/info.go

```diff
@@ -207,6 +207,10 @@ func GetReferenceInfoFile(securityContextDir string, ReferenceInfoFilename strin
 	return encodedUvmReferenceInfo, nil
 }
 
+func ThimCertsAbsent(thim *THIMCerts) bool {
+	return reflect.DeepEqual(*thim, THIMCerts{})

```

Comment:
We don't need reflect.  Just check if the len of the vcek cert and certificate chain is 0.

Reply:
done